src/bidiMapper/domains/network/NetworkRequest.ts

```diff
@@ -407,7 +414,7 @@ export class NetworkRequest {
     return {
       method: ChromiumBidi.Network.EventNames.BeforeRequestSent,
       params: {
-        ...this.#getBaseEventParams(),
+        ...this.#getBaseEventParams(Network.InterceptPhase.BeforeRequestSent),
```

Comment:
The same should be done for `network.responseStarted` event, but it looks like we don't have it implemented yet. Correct me if I'm wrong please.